api/v1alpha1/consts.go

```diff
@@ -0,0 +1,77 @@
+package v1alpha1
+
+import "time"
+
+const (
+	CertificateSigningRequestAPI      = "v1alpha1"
```

Comment:
```suggestion
	CertificateSigningRequestAPIVersion      = "v1alpha1"
```
?

Reply:
I wonder, since all those API Versions ar under api/v1alpha/consts.go 

we could just have one const APIVersion = "v1alpha1"

Reply:
That depends on whether we'd like the individual resource kinds to evolve/mature somewhat independently. IOW, can we imagine that, say, the Device resource becomes stable (v1), while the EnrollmentRequests becomes v1alpha2?

Then, is there anything to version about our APIs _on top of_ the sum of the individual resource kinds? I believe there is, e.g. common patterns on the verbs, query params, etc. If we want to have a change later to evolve that, we also need this part of the API to be versioned.

@avishayt ^?

Reply:
Could we separate the API versioning from this PR, which is just a refactoring to unblock other work?

Reply:
This PR doesn't change anything.  Let's merge and discuss options.